documentation/slate/source/includes/welcome.md

```diff
@@ -145,4 +145,4 @@ In order to report bug and make feature requests please use our github navitia p
 
 Stay tuned on twitter [@navitia](https://twitter.com/navitia).
 
-At last, we are present on IRC on the network <a href="https://webchat.freenode.net/">Freenode</a>, channel <b>#navitia</b>.
+At last, we are present on the network [`matrix.org`](https://www.matrix.org/), channel [`#navitia:matrix.org`](https://riot.im/app/#/room/#navitia:matrix.org).
```

Comment:
For clarity, I would put only one link.
```suggestion
At last, we are present on the network `matrix.org`, channel [`#navitia:matrix.org`](https://riot.im/app/#/room/#navitia:matrix.org).
```

Reply:
Done in ecdebf85e4cbff8b2a6c1f21a5a17a41f23a1567.